### PR TITLE
Version 3 needed for Pi 4 ARM64

### PR DESCRIPTION
**Pull request by manakill**
pi@raspberrypi:~/dockerfiles $ sudo docker-compose up -d
ERROR: Version in "./docker-compose.yml" is unsupported. You might be seeing this error because you're using the wrong Compose file version. Either specify a supported version (e.g "2.2" or "3.3") and place your service definitions under the `services` key, or omit the `version` key and place your service definitions at the root of the file to use version 1.
For more on the Compose file format versions, see https://docs.docker.com/compose/compose-file/

Changing to Version 3 removes this error and it starts the Containers error free

### CODE REVIEW

**Comment by lwj5** [comment]
What version of docker are you running?

check compatibility matrix here
https://docs.docker.com/compose/compose-file/compose-versioning/

**Comment by manakill** [comment]
No I didn't use the Compatibility Matrix. I used the Docker directions and the Docker image provided by IN

Client:
 Version:           18.09.1
 API version:       1.39
 Go version:        go1.11.6
 Git commit:        4c52b90
 Built:             Fri, 13 Sep 2019 10:45:43 +0100
 OS/Arch:           linux/arm64
 Experimental:      false

Server:
 Engine:
  Version:          18.09.1
  API version:      1.39 (minimum version 1.12)
  Go version:       go1.11.6
  Git commit:       4c52b90
  Built:            Fri Sep 13 09:45:43 2019
  OS/Arch:          linux/arm64
  Experimental:     false

**Comment by manakill** [comment]
Dude, that's exactly what I just posted lol.

It is compatible. IN is currently running on my R Pi 4 ARM64

**Comment by lwj5** [comment]
I do not see a docker compose version. Only a client and server version.
Could you follow the command I provided.

**Comment by manakill** [comment]
oops sorry,
docker-compose version 1.21.0, build unknown

**Comment by manakill** [comment]
It was basically a clean Pi with Raspian 64bit flashed on. I did apt update, and upgrade. Then apt installed docker and docker-compose and followed the Directions in the IN Docker git. And that brings us to now :)

**Comment by lwj5** [comment]
According to https://docs.docker.com/compose/release-notes/#1220, version 3.7 is only available to docker-compose after version 1.22.0.

I would suggest for you to install docker-ce using the guide here:
https://docs.docker.com/engine/install/ 

This will install the latest docker community (server+client) and compose and is the usual method to install docker on most OS.
The docker engine bundled with Debian buster or Raspberian is very old.

**Comment by manakill** [comment]
Ok following your steps, I got docker-ce installed instead now and 
docker --version (is updated to):
Docker version 20.10.6, build 370c289

docker-compose --version (is the same version):
docker-compose version 1.21.0, build unknown

Containers are still running fine. Either way it seems version 3 in the docker-compose.yml is a good option

I will start a fresh install, with updated directions and using docker-ce to see if the results are the same

**Comment by lwj5** [comment]
You will need to install latest docker compose as well. Please google for the install procedure. It’s available on docker website

**Comment by manakill** [comment]
I got docker-compose installed after messing with getting Python upgraded first.

**docker-compose version:**
docker-compose version 1.25.5, build unknown
docker-py version: 4.4.4
CPython version: 3.7.3
OpenSSL version: OpenSSL 1.1.1d  10 Sep 2019

**docker version:**
sudo docker version
Client: Docker Engine - Community
 Version:           20.10.6
 API version:       1.41
 Go version:        go1.13.15
 Git commit:        370c289
 Built:             Fri Apr  9 22:46:02 2021
 OS/Arch:           linux/arm64
 Context:           default
 Experimental:      true

Server: Docker Engine - Community
 Engine:
  Version:          20.10.6
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.13.15
  Git commit:       8728dd2
  Built:            Fri Apr  9 22:44:13 2021
  OS/Arch:          linux/arm64
  Experimental:     false
 containerd:
  Version:          1.4.4
  GitCommit:        05f951a3781f4f2c1911b05e61c160e9c30eaa8e
 runc:
  Version:          1.0.0-rc93
  GitCommit:        12644e614e25b05da6fd08a38ffa0cfe1903fdec
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0


The only error I got when running compose was:
pi@in:~/dockerfiles $ sudo docker-compose up -d
/usr/lib/python3/dist-packages/requests/__init__.py:91: RequestsDependencyWarning: urllib3 (1.25.11) or chardet (3.0.4) doesn't match a supported version!
  RequestsDependencyWarning)


**In Conclusion,** Pi 4 ARMv64 can run on 3.7 only if extra steps are done beforehand to make sure Python is updated. May need to be added into a new section for Prerequisites for Pi in the directions. Also found a couple more commands needing sudo. Currently running healthy containers!

**Comment by lwj5** [comment]
Good to hear. Hope they get Raspberian 64 to stable soon.

If you don’t have to run docker or docker compose for that matter with sudo, you will need to follow the guide here

https://docs.docker.com/engine/install/debian/
Under the section

> If you would like to use Docker as a non-root user
